recipes/forceatlas2-python/meta.yaml

```diff
@@ -10,37 +10,40 @@ source:
   sha256: 57bbe7d3bc74f7dd407822f58fcbd4047777451c9d694d4c4eedf378e3212b2c
 
 build:
-  noarch: generic
-  number: 0
+  number: 1
+  noarch: python
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
 
 requirements:
   host:
+    - python >=3.6
     - pandas >=0.21
     - pip
-    - python >=3.6
     - openjdk >=8
     - setuptools_scm
   run:
+    - python >=3.6
     - pandas >=0.21
     - importlib_metadata >=0.7
-    - python >=3.6
+    - setuptools
     - openjdk >=8
 
 test:
   imports:
     - forceatlas2
+  commands:
+    - java --version
 
 about:
   home: "https://github.com/klarman-cell-observatory/forceatlas2-python"
   license: "GNU General Public v3 (GPLv3)"
   license_family: GPL3
   license_file: LICENSE 
-  summary: "Multithreaded Gephi Force Atlas2 Layout algorithm in 2D and 3D, which is used to calculate the pseudo-temporal development trajectory of single-cell data."
+  summary: "Multithreaded Gephi Force Atlas2 Layout algorithm in 2D and 3D."
+  description: |
+    This tool is used to calculate the pseudo-temporal development trajectory of single-cell data.
   dev_url: "https://github.com/klarman-cell-observatory/forceatlas2"
 
 extra:
   recipe-maintainers:
     - yihming
-    - bli25broad
-    - joshua-gould
```

Comment:
I originally added the whole team to the maintainer group, while it's only me to push the real update. I don't want to put too many people there, as otherwise I'm concerned that this may delay the update to get approval from all the maintainers.

If this is not the case, I can add them back.